code/modules/reagents/chems/chems_painkillers.dm

```diff
@@ -95,14 +112,14 @@
 
 /decl/material/liquid/painkillers/affect_overdose(var/mob/living/M)
 	..()
-	if(!narcotic)
-		return
-	M.set_hallucination(120, 30)
 	SET_STATUS_MAX(M, STAT_DRUGGY, 10)
```

Comment:
Why would overdosing on non-opiate painkillers get you high? Maybe move the druggy effect under `if(narcotic)` below.